source/utilities.tex

```diff
@@ -17961,6 +18013,24 @@
 in the context of the corresponding definition
 notwithstanding the restrictions of~\ref{declval}.
 
+\pnum
+For the purpose of defining the templates in this subclause,
+let \tcode{\placeholder{VAL}<T>} for some type \tcode{T} be
+an expression defined as follows:
+\begin{itemize}
+\item
+If \tcode{T} is a reference or function type,
+\tcode{\placeholder{VAL}<T>} is an expression
+with the same type and value category as \tcode{declval<T>()}.
+\item
+Otherwise, \tcode{\placeholder{VAL}<T>} is a prvalue
+that initially has type \tcode{T}.
+\begin{note}
+If \tcode{T} is cv-qualified,
+the cv-qualification is subject to adjustment\iref{expr.type}.
+\end{note}
+\end{itemize}
+
```

Comment:
Need to add two rows to the end of [meta.unary.prop] table:
<img width="1229" alt="image" src="https://user-images.githubusercontent.com/22275534/153371077-94de47cc-e498-4e34-8c50-c045f94ebe3b.png">

Reply:
fixed